### PR TITLE
Add warning if fastlane is executed without `bundle exec`

### PR DESCRIPTION
**Pull request by KrauseFx**
This PR also updates the links to our new docs

Related https://github.com/fastlane/fastlane/issues/9748
Related https://twitter.com/orta/status/885543142405140480

### CODE REVIEW

File: fastlane/lib/fastlane/cli_tools_distributor.rb [hunk 1]
```diff
@@ -18,8 +18,8 @@ def take_off
         require "fastlane" # this might take a long time if there is no Gemfile :(
 
         # We want to avoid printing output other than the version number if we are running `fastlane -v`
-        if Time.now - before_import_time > 3 && !running_version_command?
-          print_slow_fastlane_warning
+        unless running_version_command?
```
Comment (ohayon) [inline fastlane/lib/fastlane/cli_tools_distributor.rb]: I'm not sure I understand this change, if a user has _fastlane_ installed in their gemset, and in their bundle with different versions, not using `bundle exec` will print a different version than will be executed when running with `bundle exec` right?

**Review by ohayon — CHANGES_REQUESTED** [review]
Is there something we could do in a Fastfile to say that `bundle exec` needs to be used when executing? Otherwise, if i understand this PR correctly, this change will get in the way of people using _packaged-fastlane_, or people that just don't want to have a Gemfile, right?

File: fastlane/lib/fastlane/cli_tools_distributor.rb (hunk 1 above)
Comment (KrauseFx) [inline fastlane/lib/fastlane/cli_tools_distributor.rb]: Sorry, I don't understand the question.

**Comment by KrauseFx** [comment]
> Is there something we could do in a Fastfile to say that bundle exec needs to be used when executing? Otherwise, if i understand this PR correctly, this change will get in the way of people using packaged-fastlane, or people that just don't want to have a Gemfile, right?

We've shown the warning for a while when fastlane is slow starting up, except when it's a bundled fastlane (homebrew, mac app). We encourage people to use a `Gemfile`. This PR will change the behavior that it will warn people that don't use `bundle exec`, even though they have a `Gemfile`, to use `bundle exec`.

**Comment by ohayon** [comment]
Ah! Got it!

**Review by ohayon — APPROVED** [review]

**Pull request merged**